### DISCUSSION TITLE
SFTPGo Downloads fail at ~1GB

### DISCUSSION THREAD

**Discussion by Bruellberg** [drakkan/sftpgo#1630] [Q&A]
Dear All,

SFTPGo is set up as a docker on UnRAID. Using DuckDNS and Nginx Proxy Manager, the client UI is publicly accessible.

I created a password-protected SFTPGo client account with list & download permissions, everything else is default. File downloads work very well, but large files fail to download above 1 GB.

When downloading a test file, the docker logs and the admin UI (Active Connections) indicate a file size of 1 GB although the file is actually 5.5 GB! In the client UI file manager, the file size is correctly shown...

Any idea why SFTPGo doesn't recognize big files correctly?

Thanks in advance!
Brüllberg

**Comment by schankst** [comment]
I second this. Same thing here with the latest version.  Uploads larger than 1 GB fail as well.

**Comment by dellorianes** [comment]
Same here
